apps/patient/src/components/PharmacyList.tsx

```diff
@@ -1,9 +1,16 @@
-import { Button, SlideFade, VStack } from '@chakra-ui/react'
+import { Button, SlideFade, Text, VStack } from '@chakra-ui/react'
 
 import { PharmacyCard } from '../components/PharmacyCard'
 import t from '../utils/text.json'
```

Comment:
Why are we keeping these bits of copy in this file?

Reply:
Just my preferred way of handling copy. Nice to not have long strings in code, if we ever add languages copy needs to be centralized, and helps keep grammar/tone consistent.